kivymd/uix/refreshlayout/refreshlayout.py

```diff
@@ -150,7 +152,7 @@ def on_overscroll(
             return False
 
 
-class MDScrollViewRefreshLayout(MDScrollView):
+class MDScrollViewRefreshLayout(ThemableBehavior, MDScrollView):
```

Comment:
Why did you add the ThemableBehavior class as an heir?

Reply:
In my opinion it's the most elegant solution possible here. Yeah, I could access the `ThemableBehavior.theme_cls.primary_dark` in `RefreshSpinner`, but in that case I had to  use some crutches like additional variable that can be `None` since `ColorProperty(None)` without reassignment causes crash

Reply:
You don't use `ThemableBehavior` class attributes anywhere

Reply:
I do use it in line 199 
``` py
        if not self.circle_color:
            self.circle_color = self.theme_cls.primary_dark
``` 
`theme_cls` is inherited from class `ThemableBehavior`